src/main/java/com/minecolonies/apiimp/CommonMinecoloniesAPIImpl.java

```diff
@@ -38,20 +39,21 @@
-    private        IForgeRegistry<BuildingEntry>                   buildingRegistry;
-    private final  IBuildingDataManager                            buildingDataManager    = new BuildingDataManager();
-    private final  IJobDataManager                                 jobDataManager         = new JobDataManager();
-    private final  IGuardTypeDataManager                           guardTypeDataManager   = new com.minecolonies.coremod.colony.buildings.registry.GuardTypeDataManager();
-    private        IForgeRegistry<JobEntry>                        jobRegistry;
-    private        IForgeRegistry<GuardType>                       guardTypeRegistry;
-    private        IForgeRegistry<InteractionResponseHandlerEntry> interactionHandlerRegistry;
-    private final  IInteractionResponseHandlerDataManager          interactionDataManager = new InteractionResponseHandlerManager();
-    private        IForgeRegistry<ColonyEventTypeRegistryEntry>    colonyEventRegistry;
-    private static IGlobalResearchTree                             globalResearchTree     = new GlobalResearchTree();
+    private final  IColonyManager                                          colonyManager          = new ColonyManager();
+    private final  ICitizenDataManager                                     citizenDataManager     = new CitizenDataManager();
+    private final  IMobAIRegistry                                          mobAIRegistry          = new MobAIRegistry();
+    private final  IPathNavigateRegistry                                   pathNavigateRegistry   = new PathNavigateRegistry();
+    private        IForgeRegistry<BuildingEntry>                           buildingRegistry;
+    private final  IBuildingDataManager                                    buildingDataManager    = new BuildingDataManager();
+    private final  IJobDataManager                                         jobDataManager         = new JobDataManager();
+    private final  IGuardTypeDataManager                                   guardTypeDataManager   = new com.minecolonies.coremod.colony.buildings.registry.GuardTypeDataManager();
+    private        IForgeRegistry<JobEntry>                                jobRegistry;
+    private        IForgeRegistry<GuardType>                               guardTypeRegistry;
+    private        IForgeRegistry<InteractionResponseHandlerEntry>         interactionHandlerRegistry;
+    private final  IInteractionResponseHandlerDataManager                  interactionDataManager = new InteractionResponseHandlerManager();
+    private        IForgeRegistry<ColonyEventTypeRegistryEntry>            colonyEventRegistry;
+    private        IForgeRegistry<ColonyEventDescriptionTypeRegistryEntry> colonyEventDescriptionRegistry;
+    private static IGlobalResearchTree                                     globalResearchTree     = new GlobalResearchTree();
```

Comment:
What happened here?

Reply:
formatting changes happened here, there were spaces to make all the names start at the same point, and since the new one has a longer type i moved all the var names.

Reply:
Okey